src/lib.rs

```diff
@@ -801,10 +830,11 @@ pub struct Closed01<F>(pub F);
 
 /// The standard RNG. This is designed to be efficient on the current
 /// platform.
+///
+/// The underlying algorithm is not fixed, thus values from this generator
+/// cannot be guaranteed to be reproducible.
 #[derive(Clone, Debug)]
-pub struct StdRng {
-    rng: IsaacWordRng,
-}
+pub struct StdRng(IsaacWordRng);
```

Comment:
Maybe I should not have changed `StdRng` in this PR, but it seemed like a minor change and made a direct copy of the rest possible.